wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py

```diff
@@ -15,36 +14,34 @@
 """
 
 
-def startServer(conf='odm2_config_timeseries.cfg', openPort=8080):
-
+def get_connection(conf):
     # Parse connection from config file
     config = configparser.ConfigParser()
     with open(conf, 'r') as configfile:
         config.read_file(configfile)
         connection = config['Database']['Connection_String']
+        return connection
```

Comment:
Just to be on the safe side I would  place the `return` out of the `with` block.